.gitignore

```diff
@@ -25,3 +25,6 @@ Cargo.lock
 
 # final wit builds
 *.wasm
+
+wit-definitions/aggregator/wit/deps
```

Comment:
how are these generated in the first place? in any case - if we're doing this, should probably be for `types/` too...